tests/unit/helpers/index.spec.js

```diff
@@ -0,0 +1,9 @@
+import { setLength } from "@/helpers";
+
+  describe('setLength', () => {
+      it('returns values in hours and minutes', () => {
+          const itemInMinutes = 125;
+          const output = `${2}h ${5} min`
```

Comment:
a little thing: why not just: 
```js
const output = '2h 5 min'
```

btw. spaces are not consistent - maybe you should tweak a bit the function?